javax-money/src/main/java/com/fasterxml/jackson/datatype/money/MoneyModule.java

```diff
@@ -0,0 +1,171 @@
+package com.fasterxml.jackson.datatype.money;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.core.util.VersionUtil;
+import com.fasterxml.jackson.databind.Module;
+import com.fasterxml.jackson.databind.module.SimpleDeserializers;
+import com.fasterxml.jackson.databind.module.SimpleSerializers;
+import org.apiguardian.api.API;
+import org.javamoney.moneta.FastMoney;
+import org.javamoney.moneta.Money;
+import org.javamoney.moneta.RoundedMoney;
+
+import javax.money.CurrencyUnit;
+import javax.money.MonetaryAmount;
+import javax.money.MonetaryOperator;
+import javax.money.MonetaryRounding;
+import javax.money.format.MonetaryFormats;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+import static org.apiguardian.api.API.Status.STABLE;
+
+
+@API(status = STABLE)
+public final class MoneyModule extends Module {
+
+    private final AmountWriter<?> writer;
+    private final FieldNames names;
+    private final MonetaryAmountFormatFactory formatFactory;
+    private final MonetaryAmountFactory<? extends MonetaryAmount> amountFactory;
+    private final MonetaryAmountFactory<FastMoney> fastMoneyFactory;
+    private final MonetaryAmountFactory<Money> moneyFactory;
+    private final MonetaryAmountFactory<RoundedMoney> roundedMoneyFactory;
+
+    public MoneyModule() {
+        this(new DecimalAmountWriter(), FieldNames.defaults(), MonetaryAmountFormatFactory.NONE,
+                Money::of, FastMoney::of, Money::of, RoundedMoney::of);
+    }
+
+    private MoneyModule(final AmountWriter<?> writer,
+                        final FieldNames names,
+                        final MonetaryAmountFormatFactory formatFactory,
+                        final MonetaryAmountFactory<? extends MonetaryAmount> amountFactory,
+                        final MonetaryAmountFactory<FastMoney> fastMoneyFactory,
+                        final MonetaryAmountFactory<Money> moneyFactory,
+                        final MonetaryAmountFactory<RoundedMoney> roundedMoneyFactory) {
+
+        this.writer = writer;
+        this.names = names;
+        this.formatFactory = formatFactory;
+        this.amountFactory = amountFactory;
+        this.fastMoneyFactory = fastMoneyFactory;
+        this.moneyFactory = moneyFactory;
+        this.roundedMoneyFactory = roundedMoneyFactory;
+    }
+
+    @Override
+    public String getModuleName() {
+        return MoneyModule.class.getSimpleName();
+    }
+
+    @Override
+    @SuppressWarnings("deprecation")
+    public Version version() {
```

Comment:
This is unnecessarily using `VersionUtil` instead of referencing generated `PackageVersion` and doing

```
public Version version() {
    return PackageVersion.VERSION;
}
```